### DISCUSSION TITLE
Rendering on new computer mucks formatting up - help!

### DISCUSSION THREAD

**Discussion by pgseye** [quarto-dev/quarto-cli#8494] [Q&A] [answered]
### Description

I recently updated my 8 yr old iMac to a Mac Studio. I opened up and rendered a talk I had been working on and the image positioning in some slides is mucked up (see attached examples of old and new computer). The thing is - the code is completely unchanged and everything looks correct as it did before. I can render this same .qmd file on my 4 yr old MacBook Pro and the images look correct (as they did on my iMac). 

I am using Quarto v. 1.0.28 on my MacBook and it looks like 1.3.450 on my new Mac Studio.

I don't know how to troubleshoot this when the code is unchanged and everything works as it should on one machine. Any ideas where to start as this is pretty frustrating? It's not an intel/Apple silicon thing is it?

Thank you - Paul

The code I used to produce this in the slide is:

```
## Some basic concepts

<br>

These all play a role in logistic regression, but they are nothing more than transformations of one another:

::: {.column width="33.3%"}
::: {.fragment .fade-in}
<br>

::: {style="font-size: 1.2em"}
Probability
:::
:::

::: {.fragment .fade-in}
![](images/prob_vert.png){.absolute height="300" left="80"}
:::
:::

::: {.column width="33.3%"}
::: {.fragment .fade-in}
<br>

::: {style="font-size: 1.2em"}
Odds
:::
:::

::: {.fragment .fade-in}
![](images/odds_vert.png){.absolute height="300" left="420"}
:::
:::

::: {.column width="33.3%"}
::: {.fragment .fade-in}
<br>

::: {style="font-size: 1.2em"}
log-odds
:::
:::

::: {.fragment .fade-in}
![](images/logodds_vert.png){.absolute height="300" left="770"}
:::
:::

::: fragment
![](images/arrows.png){.absolute top="320" left="0"}
:::
```
<img width="1909" alt="screenshot_02" src="https://github.com/quarto-dev/quarto-cli/assets/12015304/c6c42d53-22b0-4426-921c-5298e6e26942">
<img width="1891" alt="screenshot_03" src="https://github.com/quarto-dev/quarto-cli/assets/12015304/57898e36-a97c-4850-9552-47ef6dd5c2d1">

**Comment by pgseye** [comment]
I have created this as an issue and am happy to delete this as a discussion but can’t see a way to do that.

**Reply by mcanouil** [reply]
No needs to delete anything.
If there is a bug, an issue can be created out of the discussion by a single click on a "button"/link.

**Comment by mcanouil** [comment, marked as answer]
Closing this as outdated since an issue was opened:
- https://github.com/quarto-dev/quarto-cli/issues/8498
